.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 100,
+  "trailingComma": "all",
```

Comment:
Personally I would go with `es5` but we're going with a pretty recent version of node for source code anyway so... doesn't really matter

Reply:
This isn't a hill I'd die on, but I would sustain minor wounds defending it.

If you have aesthetic or functional reasons for preferring no trailing commas, I'm happy to hear them. Compatibility isn't an argument I buy anymore. Node [has supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#Browser_compatibility) most kinds of trailing commas since pre-1.0, and trailing commas in function arguments since 8.0. Browsers have similar levels of support.

Reply:
Yeah since we're transpiling it anyways, really we're targeting the TS compiler and not the runtime, and the TS compiler is happy with trailing commas and I think they make a big difference for maintainability for I'm pro trailing commas